terraform/ec2/linux/install-efs-utils.sh

```diff
@@ -0,0 +1,32 @@
+# install amazon EFS client on linux, agnostic to yum or apt
```

Comment:
We already talked about this separately, but we should update our AMIs that we build for testing to install the EFS utils instead of doing it in our terraform for test setup